packages/ember-glimmer/lib/ember-template-compiler/plugins/old-syntax.js

```diff
@@ -0,0 +1,35 @@
+import { assert } from 'ember-metal/debug';
```

Comment:
I'd rather put this in `packages/ember-template-compiler` instead of nesting it within the `ember-glimmer` package.  I'm sure there is a reason for doing it here for now, can you recall what it is?